src/main/resources/org/broadinstitute/hellbender/utils/plotter/CNV_plotting_library.R

```diff
@@ -47,8 +47,36 @@ SetUpPlot = function(y.lab, y.min, y.max, x.lab, label_chromosomes, num_chromoso
   }
 }
 
+## Chromosome Background for data
+SetUpPlotPerSeg = function(y.lab, y.min, y.max, x.lab, label.x, x.axt, segs){
+  #plot in megabases
+  scaling = 1000000
+  start = segs[1, "Start"]
+  end = segs[nrow(segs), "End"]
+  chr = segs[1,"Chromosome"]
+  suppressWarnings( par( mar=c(3.1,3.6,0.1,0),mgp=c(2,-0.2,-1.1) ) )
+  options(scipen=3)
+  plot(1, type="n", xlab="", ylab="", xlim=c(start, end), ylim=c(y.min, y.max), bty='n', xaxt='n')
+  scaled_start = start/scaling
+  scaled_end = end/scaling
+  labels=signif(seq(scaled_start, scaled_end, by=((scaled_end-scaled_start)/10)), 2)
+  sigs = 0
```

Comment:
I think I get what's happening here --  incrementing the number of significant figures until you reach a precision at which all the axis labels are unique.  This should be done without a loop i.e.
```java
increment = (scaled_end - scaled_start)/10
num_sig_figs = [something related to the size of the increment]
labels = signif(seq(scaled_start, scaled_end, by=increment), num_sig_figs)
```